__tests__/models/transaction.test.js

```diff
@@ -51,6 +93,27 @@ test('Token tx', () => {
   expect(info.length).toBe(5);
   expect(info[2].toString('hex')).toBe('54657374');
   expect(info[4].toString('hex')).toBe('545354');
+
+  const address1 = new Address('WR1i8USJWQuaU423fwuFQbezfevmT4vFWX');
+  const output1 = new Output(1000, address1);
+  const inputDataHex = '4630440220317cd233801c1986c2de900bf8d344c6335d3c385e69d19d65e1fae7a0afd0af02207acddb824debf855798d79c45701cbe3a19aea00baad94bff5290c6f0b0acf8e210346cddff43dffab8e13398633ab7a7caf0d634551e89ae6fd563e282f6744b983';
```

Comment:
I feel you should add some junk sequence of bytes to confirm it fails the right way.

Reply:
Done.